packages/dmn-js-decision-table/test/spec/features/column-resize/ColumnResizeSpec.js

```diff
@@ -0,0 +1,169 @@
+      ModelingModule,
+      ColumnResizeModule
+    ],
+    debounceInput: false
+  }));
+
+  let testContainer;
+
+  beforeEach(function() {
+    testContainer = TestContainer.get(this);
+  });
+
+
+  it('should resize input clause', function() {
+
+    // given
+    const inputHeader = domQuery('.input-cell', testContainer);
+    const resizeHandle = domQuery('.resize-column-handle', inputHeader);
+    const initialWidth = widthAsNumber(getComputedStyle(inputHeader).width);
+
+    const initialX = resizeHandle.getBoundingClientRect().left + 10;
+
+    // when
+    triggerMouseEvent(resizeHandle, 'mousedown', initialX, 0);
+    triggerMouseEvent(document, 'mousemove', initialX + 50, 0);
+    triggerMouseEvent(document, 'mouseup', initialX + 50, 0);
+
+    // then
+    const finalWidth = widthAsNumber(getComputedStyle(inputHeader).width);
+
+    expect(finalWidth).to.be.eql(initialWidth + 50);
+  });
+
+
+  it('should NOT resize input clause below 150px', function() {
+
+    // given
+    const inputHeader = domQuery('.input-cell', testContainer);
+    const resizeHandle = domQuery('.resize-column-handle', inputHeader);
+
+    const initialX = resizeHandle.getBoundingClientRect().left + 10;
+
+    // when
+    triggerMouseEvent(resizeHandle, 'mousedown', initialX, 0);
+    triggerMouseEvent(document, 'mousemove', initialX - 50, 0);
+    triggerMouseEvent(document, 'mouseup', initialX - 50, 0);
+
+    // then
+    const finalWidth = widthAsNumber(getComputedStyle(inputHeader).width);
+
+    expect(finalWidth).to.be.eql(150);
+  });
+
+
+  it('should resize output clause', function() {
+
+    // given
+    const outputHeader = domQuery('.output-cell', testContainer);
+    const resizeHandle = domQuery('.resize-column-handle', outputHeader);
+    const initialWidth = widthAsNumber(getComputedStyle(outputHeader).width);
+
+    const initialX = resizeHandle.getBoundingClientRect().left + 10;
+
+    // when
+    triggerMouseEvent(resizeHandle, 'mousedown', initialX, 0);
+    triggerMouseEvent(document, 'mousemove', initialX + 50, 0);
+    triggerMouseEvent(document, 'mouseup', initialX + 50, 0);
+
+    // then
+    const finalWidth = widthAsNumber(getComputedStyle(outputHeader).width);
+
+    expect(finalWidth).to.be.eql(initialWidth + 50);
+  });
+
+
+  it('should NOT resize output clause below 150px', function() {
+
+    // given
+    const outputHeader = domQuery('.output-cell', testContainer);
+    const resizeHandle = domQuery('.resize-column-handle', outputHeader);
+
+    const initialX = resizeHandle.getBoundingClientRect().left + 10;
+
+    // when
+    triggerMouseEvent(resizeHandle, 'mousedown', initialX, 0);
+    triggerMouseEvent(document, 'mousemove', initialX - 50, 0);
+    triggerMouseEvent(document, 'mouseup', initialX - 50, 0);
+
+    // then
+    const finalWidth = widthAsNumber(getComputedStyle(outputHeader).width);
+
+    expect(finalWidth).to.be.eql(150);
+  });
+
+
+  it('should resize annotation', function() {

```

Comment:
Maybe just a detail, but we are introducing this behavior with the next commit (https://github.com/bpmn-io/dmn-js/pull/530/commits/ab8718a17327fbb51b3567a7b640aecd643d79ac), right? It is currently located in another (https://github.com/bpmn-io/dmn-js/pull/530/commits/2e3262575cf5b45931a038f3079f2d6d952c3598)

Reply:
Good catch.

Reply:
I squashed the input/output resizing with general column resize. The tests for annotations are now in the annotations commit.